packages/jest/package.json

```diff
@@ -35,18 +35,19 @@
     "migrations": "./migrations.json"
   },
   "dependencies": {
-    "@jest/reporters": "^29.4.1",
-    "@jest/test-result": "^29.4.1",
+    "@jest/reporters": "^29.4.1 || ^30.0.0-alpha.6",
```

Comment:
These are our `dependencies`... so this would pretty much resolve `30.0.0-alpha6` right? People shouldn't really care.. but should we depend on Jest's alpha?

Reply:
That's a good point. At least `pnpm` resolves one or the other depending on the version that gets resolved from other package dependencies or the repo dependencies. I haven't tested with the other package managers. Still, I agree that's better to be explicit and not depend on the alpha. Given these are dependencies, they shouldn't cause any conflicts with whatever the user has.